.github/workflows/generate-package-metadata.yml

```diff
@@ -30,13 +30,6 @@ jobs:
         run: echo ${{ matrix.repoSlug }}
       - name: SchemaFile
```

Comment:
We don't need these echo statements - this was purely for initial testing that my matrix worked :)